### PR TITLE
#969: Docs - Impliment Alerts

### PR DESCRIPTION
**Pull request by saadmhybris**
#969

### CODE REVIEW

File: test/templates/icon/component.njk [hunk 1]
```diff
@@ -4,5 +4,5 @@
     instead of more complex modifier.block options
 #}
 {%- macro icon(glyph="", size="", classes="") -%}
-<span class="tn-icon{{ (glyph if size == "" else [glyph, size]) | modifier('icon') }}{{ classes | classes }}" role="presentation"></span>
```
Comment (xak) [inline test/templates/icon/component.njk]: This should stay. I added this for the dropdown based on what we had discussed. Removing it breaks the dropdown component.

Comment (saadmhybris) [reply test/templates/icon/component.njk]: this is adding an empty class "tn-"

<span class="tn-icon tn-icon--add tn-" role="presentation"></span>

Comment (xak) [reply test/templates/icon/component.njk]: This should be fixed with #978 — https://github.com/SAP/techne/pull/978/files#diff-ace3b50701c30da220db72a852e66f79R52

Comment (saadmhybris) [reply test/templates/icon/component.njk]: ok will restore and merge.

**Pull request merged**